### PR TITLE
feat: update schemas

### PR DESCRIPTION
**Pull request by ronenkapelian**
Adding more columns to polygon parts table

* imageName,  ProductType
* adding insert query with geojson geom (insted of wkt)

* update fullScheme creation script

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                       |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/DAL/migration/FullSchema.sql [hunk 1]
```diff
@@ -24,6 +22,8 @@ CREATE TABLE IF NOT EXISTS "PolygonParts".parts
     classification numeric NOT NULL,
     description text COLLATE pg_catalog."default",
     geom geometry(Polygon,4326) NOT NULL,
+    "imageName" text COLLATE pg_catalog."default",
+    "productType" text COLLATE pg_catalog."default" NOT NULL DEFAULT 'orthophotoBest'::text,
```
Comment (asafMasa) [inline src/DAL/migration/FullSchema.sql:26]: the default should be in code and not in scheme

File: src/DAL/migration/insertDataExample.sql [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+	VALUES ('1328b7b4-e4e5-4d7f-a00f-087a2fab6309','MosaicBest','orthphoto','1.0', '2022-10-10 11:30:30', '2022-10-10 11:30:30', 0.000000335276126861572, 0.072, 5, 5, 10, '1, 2, 3', 'world', 5, 'some example', ST_GeometryFromText('POLYGON((-180 -90,-180 90,180 90,180 -90,-180 -90))'), 'test', 'best');
+
+
+-- Example with geojson geometry
+
+INSERT INTO "PolygonParts".parts(
+	"recordId", "productId", "productName", "productVersion", "sourceStartDateUtc", "sourceEndDateUtc", "minResolutionDegree", "maxResolutionDegree", "minResolutionMeter", "maxResolutionMeter", "minHorizontalAccuracyCe90", sensors, region, classification, description, geom, "imageName", "productType")
+	VALUES ('1328b7b4-e4e5-4d7f-a00f-087a2fab6309', 'MosaicBest', 'orthophoto', '1.0', '2022-10-10 11:30:30', '2022-10-10 11:30:30', 0.000000335276126861572, 0.072, 5, 5, 10, '1, 2, 3', 'world', 5, 'some example', ST_GeomFromGeoJSON('{"coordinates": [[[-180, -90],[-180,90],[180,90],[180,-90],[-180,-90]]],"type": "Polygon"}'), 'test', 'best');
```
Comment (asafMasa) [inline src/DAL/migration/insertDataExample.sql:14]: add space at the end of the file

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: src/DAL/migration/insertDataExample.sql (hunk 2 above)
Comment (ronenkapelian) [reply src/DAL/migration/insertDataExample.sql:14]: added

**Review by asafMasa — APPROVED** [review]

**Pull request merged**